resources/js/main.js

```diff
@@ -76,8 +76,13 @@ function updateDevices() {
         .catch((e) => console.error(e));
 }
 
+function displaySelectedDevice(uuid){
+    document.getElementById('targetDeviceLabel').value = document.getElementById(uuid).innerHTML;
+}
+
 function selectDevice(event, uuid) {
     target.uuid = uuid;
+    displaySelectedDevice(uuid);
```

Comment:
I'd keep `document.getElementById('targetDeviceLabel').value = document.getElementById(uuid).innerHTML;` here instead of creating a function for it. It's also the only event that causes `#targetDeviceLabel.value` to change.

Reply:
Sounds good!
changed https://github.com/pagopa/push-notificator/pull/5/commits/59042b7e51463c3dc24462db2437039d8acaef5d